packages/block-library/src/blocks/hero/preview.js

```diff
@@ -6,7 +6,9 @@ import HeroBackground from './background';
 /**
  * WordPress dependencies
  */
-import { InnerBlocks } from '@wordpress/block-editor';
+import {
+  __experimentalUseInnerBlocksProps as useInnerBlocksProps
```

Comment:
Maybe we could be more defensive using experimental features. The moment the feature is deemed not experimental anymore this will crash.

We could do something like
```js
const useInnerBlocksProps = wp.blockEditor.useInnerBlocksProps || wp.blockEditor.__experimentalUseInnerBlocksProps;
```
And finally we can check if `useInnerBlocksProps` is undefined and offer some sort of fallback in that case.